### PR TITLE
September 9th Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Changes

Release Notes for September 9th.

## Checklist for contributors

Before submitting this PR, please make sure:

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

File: content/en/docs/amplify_relnotes/20220909_marketplace.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+The following enhancements and bug fixes are available in this release.
+
+#### Feature updates for Product Foundry
+
+* **Category management** - Categories can now be edited, deleted and previewed from a dedicated screen at **Product Foundry > Categories**.
+The preview helps admnistrator to change the category visibility as well as if a cateory is featured or not.
+
+#### Bug fixes for Product Foundry
+
+* **Assigning category to product removes many product details** - Previously when creating a category and assigning product(s) to it, the assigned product(s) changed to Draft status. Now, the category assignment does not create a product Draft release anymore.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20220909_marketplace.md:113]: **Assigning category to product removes many product details** to be replaced with: **Assigning a product to a category changed the product state** - Previously when creating a category from the category management screen, and assigning product (s) to it, the assigned product (s) changed to Draft status. Now, the category assignment does not create a product Draft release.

File: content/en/docs/amplify_relnotes/20220909_marketplace.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+## Consumer experience
+
+### Marketplace
+
+The following enhancements and bug fixes are available in this release.
+
+#### Feature updates for Marketplace
+
+* **Marketplace home page - filter per featured category** - The featured categories displayed in the home page allows the user to see all related product when using the "View products" link on a category.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20220909_marketplace.md:132]: Replace *Marketplace home page - filter per featured category** with **View products in a featured category** - Consumers can now see the products included in the featured categories displayed on the home page, by clicking on the "View products" link shown for each category.

File: content/en/docs/amplify_relnotes/20220909_marketplace.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+* **Markeplace browse product - filter per category** - the browse home page is enriched with the category filter and allow user to see all product for a selected category. A breadcrumb is added as soon as a category is selected in the result page. This breadcrumb allows the user to unselect a previously selected category.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20220909_marketplace.md:133]: Replace **Markeplace browse product - filter per category**  with **Filter products by categories** - the browse products page has been enhanced with a category filter to allow users to filter the catalog of products by categories.  A breadcrumb is added to the result page once the filter in applied to enable users to unselect a selected category and reset the filter.

File: content/en/docs/amplify_relnotes/20220909_marketplace.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+
+#### Bug fixes for Marketplace
+
+* **Paid plan base price with large number crash the backend** - Previously when using a big number in paid plan, the backend api return a 500 error. Now, the UI does not allow to enter a number having more than 10 digits.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20220909_marketplace.md:137]: crashed

**Review by georgiana-caras — CHANGES_REQUESTED** [review]

File: content/en/docs/amplify_relnotes/20220909_marketplace.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+
+* **Issue** - Text.
+
+### Product Foundry
+
+The following enhancements and bug fixes are available in this release.
+
+#### Feature updates for Product Foundry
+
+* **Category management** - Categories can now be edited, deleted and previewed from a dedicated screen at **Product Foundry > Categories**.
```
[the remark this answers is not in the document]
Comment (lbadenhop) [reply content/en/docs/amplify_relnotes/20220909_marketplace.md]: updated

**Comment by achoudhary-axway** [comment]
@lbadenhop  and @sallylanda6 - I took a look on the preview but don't see anything mentioned for Business Insights / Subscriptions.

**Comment by achoudhary-axway** [comment]
@lbadenhop  and @sallylanda6 - I took a look but don't see any content for Business Insights / Subscription.

image

File: content/en/docs/amplify_relnotes/20220909_marketplace.md [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+* **Assigning a product to a category changed the product state** - Previously, when creating a category from the category management screen, and assigning product(s) to it, the assigned product(s) changed to Draft status. Now, the category assignment does not create a product Draft release.
+* **Marketplace active subscription list is empty** - Previously, it was possible to create/activate a plan with a quota having no resource assigned. Now, the quota cannot be created/updated if it has no resource assigned and a plan cannot be activated if it has no quota.
+
+#### Known issues for Product Foundry
+
+There are no known limitations in this release.
+
+### Business insights
+
+There are no enhancements and bug fixes in this release.
```
Comment (achoudhary-axway) [inline content/en/docs/amplify_relnotes/20220909_marketplace.md:121]: Subscriptions - API Providers can now see the usage and quota compliance of Subscriptions to their APIs by different Consuming Teams.

**Review by robertengaxway — APPROVED** [review]

**Review by georgiana-caras — APPROVED** [review]

**Pull request merged**